Scanner Changes (#6193)
endless-sky/endless-sky: data/tooltips.txt

	`Lets you scan your target ship's cargo from up to this distance away.`

tip "cargo scan power:"

> 	`The range of this cargo scanner is proportional to the square root of this value. At increasing distances, the scanner's speed will naturally decline.`

Reviewer: Although distance to target does decrease scan speed, I think factors that effect speed should only be mentioned in the speed tooltip.
```suggestion
	`The range of this cargo scanner is proportional to the square root of this value.`
```

> 	`The range of this cargo scanner is proportional to the square root of this value. The speed of the scanner also decreases in proportion to the square root of this value.`

Reviewer 2: Why does power affect speed? Does this mean that if we want two scanners to have different powers but the same actual speed, we have to do a weird balancing act where we adjust the speed faster as the power goes up to maintain the same actual speed?

For that matter, is this decrease corresponding to a decrease in the square root? Or to an increase in the square root?

Author: An artifact of an earlier form of the PR, most likely. Speed is uncoupled from range in this build, but speed decreases with distance (and so naturally if you expect higher ranges you'll spend more time scanning)

I'll fix the tips to the best of my ability, but I'm actually not very good at writing them. I think in a specific way.

Author: As an example of artifacts from previous builds- scanner strength no longer depends on relative velocity to your target. I really should fix these tips.

tip "cargo scan speed:"

> 	`The higher this value, the faster a scan will complete. Multiple factors can impact the speed of a scan, like the size of the target ship or its distance to the scanner.`

Reviewer: ```suggestion
	`The maximum speed of this cargo scanner is proportional to the square root of this value. The observed scan speed will be impacted by multiple factors. Targets which are farther away or have a larger cargo hold will take longer to scan.`
```

> 	`The time it takes for this scanner to complete, all things equal, is inversely proportional to this value. Multiple factors can impact the speed of a scan, like the size of the target ship or its distance to the scanner.`

Reviewer 2: Definitely clearer than before, although it might be better yet to have a simpler lead off.
```suggestion
	`The higher this value, the faster a scan will complete. The time it takes for this scanner to complete, all things equal, is inversely proportional to this value. Multiple factors can impact the speed of a scan, like the size of the target ship or its distance to the scanner.`
```

Author: I think it may be better to rapidly prototype tooltips on discord, so we can quickly settle on something good without the obligatory formality of github. I'm thinkin', like...

`The higher this value, the faster a scan will complete. Multiple factors can impact the speed of a scan, bla bla bla etc` because if it's too long of a tooltip it may be difficult to parse out. (even if the longest tooltip takes less than 5 seconds to read out. humans simply have low tolerance for visually-intimidating text.)

	`Lets you scan your target ship's outfits from up to this distance away.`

tip "outfit scan power:"

> 	`The range of this outfit scanner is proportional to the square root of this value. At increasing distances, the scanner's speed will naturally decline.`

Reviewer: ```suggestion
	`The range of this outfit scanner is proportional to the square root of this value.`
```

> 	`The range of this outfit scanner is proportional to the square root of this value. The speed of the scanner also decreases in proportion to the square root of this value.`

Reviewer 2: Like with cargo, does this mean that as power goes up, speed goes down?

tip "outfit scan speed:"

> 	`The higher this value, the faster a scan will complete. Multiple factors can impact the speed of a scan, like the size of the target ship or its distance to the scanner.`

Reviewer: ```suggestion
	`The maximum speed of this outfit scanner is proportional to the square root of this value. The observed scan speed will be impacted by multiple factors. Targets which are farther away or have a larger outfit capacity will take longer to scan.`
```

> 	`The time it takes for this scanner to complete, all things equal. is inversely proportional to this value. Multiple factors can impact the speed of a scan, like the size of the target ship or its distance to the scanner.`

Reviewer 2: ```suggestion
	`The higher this value, the faster a scan will complete. The time it takes for this scanner to complete, all things equal. is inversely proportional to this value. Multiple factors can impact the speed of a scan, like the size of the target ship or its distance to the scanner.`
```
